velox/vector/ComplexVector.cpp

```diff
@@ -528,9 +528,23 @@ void ArrayVectorBase::copyRangesImpl(
 
 void RowVector::validate(const VectorValidateOptions& options) const {
   BaseVector::validate(options);
+  std::optional<vector_size_t> lastSetBit;
```

Comment:
maybe call it, lastNonNullRowIndex

Reply:
vector_size_t lastSetBit =0

Reply:
if null is not allocated then lastSetBit should be = vector size